fishgen-py/src/lib.rs

```diff
@@ -48,6 +48,46 @@ impl From<fishgen_rs::FishOutput> for PyFishOutput {
     }
 }
 
+impl From<fishgen_rs::FishOutput> for PyImageOutput {
+    fn from(fish: fishgen_rs::FishOutput) -> Self {
+        let raw_bytes = fish.image.as_raw().iter().map(|v| *v as f32 / 255.0).collect();
+        let width = fish.image.width() as usize;
+        let height = fish.image.height() as usize;
+        let new = Array::from_shape_vec((width, height, 3), raw_bytes).unwrap();
+
+        PyImageOutput {
+            image: new.into(),
+            x: fish.x as f32 / width as f32,
+            y: fish.y as f32 / height as f32,
+            width: fish.width as f32 / width as f32,
+            height: fish.height as f32 / height as f32,
+        }
+    }
+}
+
+#[pyclass(name = "image_output")]
+pub struct PyImageOutput {
+    pub image: Array3<f32>,
+    #[pyo3(get)]
+    pub x: f32,
+    #[pyo3(get)]
+    pub y: f32,
+    #[pyo3(get)]
+    pub width: f32,
+    #[pyo3(get)]
+    pub height: f32,
+}
+#[pymethods]
+impl PyImageOutput {
+    #[getter]
+    pub fn image<'py, 'b>(&self, m: Python<'py>) -> PyResult<&'b numpy::PyArray3<f32>>
+    where
+        'py: 'b,
+    {
+        Ok(self.image.to_pyarray(m))
+    }
+}
+
```

Comment:
Not entirely sure why a different struct is needed for something that holds the same data. Perhaps a global renaming of `FishOutput` to `ImageOutput` and adding an enum for the image type?

Reply:
My intention was just to not restructure everything, but if you're okay with it then that sounds good :)